controller/prompt_generator/generate_prompt.py

```diff
@@ -0,0 +1,55 @@
+import random
+from typing import List
+
+
+class PromptGenerator:
+    """
+   A class used to generate prompts for GPT-3.
+
+   Attributes:
+       question_templates (list of str): A list of question templates for GPT-3 prompts.
+
+   Methods:
+       generate_prompt(slide_text: List[str]) -> str: Generates a prompt for GPT-3 based on the content of a slide.
+   """
+    def __init__(self):
+        self.question_templates = [
```

Comment:
very nice, but as it is static why put it in the init function?
looks as if it could be a constant.